scripts/package.json

```diff
@@ -5,9 +5,5 @@
   "type": "module",
   "scripts": {
     "test": "true"
-  },
-  "devDependencies": {
```

Comment:
The intention is to keep the root's package.json file minimal. In my opinion, it's okay to keep `globby` and `mustache` dependencies here because they are not used anywhere else, and these scripts aren't vital for WF functioning. 

What do you think?

Reply:
> these scripts aren't vital for WF functioning

I think that's the purpose of `devDependencies`.

What are the benefits of a minimal root `package.json`?

If these dependencies are kept in `scripts/package.json`, would you be open to a PR to add `scripts/yarn.lock`?

I think additionally, `scripts/package.json` would then also replicate some of the stuff in `/package.json` which standardises tools, specifically:

```json
{
  "engines": {
    "node": "^18.20.5 || ^20.18.1 || ^22.12.0",
    "yarn": "1.22.22"
  },
  "volta": {
    "node": "22.12.0",
    "yarn": "1.22.22"
  },
  "packageManager": "yarn@1.22.22"
}
```

Reply:
Alternatively, maybe `scripts/package.json` can be removed completely.

Reply:
I like the idea of adding `engines`, `volta`, and a lock file to /scripts, which will ensure consistency and maintain isolation and separation from the source. I can prepare a PR with those changes :)